README.md

```diff
@@ -1,2 +1,27 @@
 # crudApp
 I am learning from scratch how to build an application with react using crud , routing and environment set up
+
+
+I am going to add my TOdO for joining into IBM
+
+TODO:
+I should be familiar with github
+
+Learnt on 09/sep
+I have learnt to fork the project
+I have learnt to clone the project using command git clone <url>
+Next i have to create a branch using the command.
+    git checkout -b <branchname> <copyfrom the branch>
+    ex: git checkout -b addingPause  master
+List of commands available in git link: https://git-scm.com/doc
+
+Commands:-------------------------------------------
+git clone <url> ---> for cloning to the local directory
+git checkout -b addingPauseReq master ----> creating a branch in my repository and it is taking `master` as a reference to create a branch
+git status --->>> to see the added and updated files info
+git add . / git add Readme ---> trying to add a file or code or folder 
+git commit -m "message"   ----->>> commiting 
+git push origin addingPauseReq     ---->>> It will push the code tp branch addingPauseReq
+git reset --hard  ---->>> will save the previous files.
+git status
```

Comment:
```suggestion
git status --->> will show the list of status files 
```

Reply:
@krishna63 : The above command is self explanatory, i dont think we require further explanation